superset/config.py

```diff
@@ -55,7 +55,7 @@
 # ---------------------------------------------------------
 PACKAGE_DIR = os.path.join(BASE_DIR, "static", "assets")
 VERSION_INFO_FILE = os.path.join(PACKAGE_DIR, "version_info.json")
-PACKAGE_JSON_FILE = os.path.join(BASE_DIR, "assets" "package.json")
+PACKAGE_JSON_FILE = os.path.join(BASE_DIR, "assets", "package.json")
```

Comment:
Oh wow. Seems like this was probably an issue before. The infamous auto/implicit concat. I'm guessing we probably have packages with a weird `assetspackage.json` filename. Still works but would looks odd.

Reply:
Looks like it was my bad here: https://github.com/apache/incubator-superset/pull/8196 :(